src/test/java/lotto/model/wrapper/LottoNumberTests.java

```diff
@@ -18,12 +16,19 @@ public void generateLottoNumberTest(int input) {
         assertThatCode(() -> LottoNumber.of(input)).doesNotThrowAnyException();
     }
 
+    @DisplayName("로또 번호 비교 테스트")
+    @ParameterizedTest
+    @ValueSource(ints = {1, 17, 45})
+    public void compareLottoNumberTest(int input) {
+        assertThat(LottoNumber.of(input)).isEqualTo(LottoNumber.of(input));
```

Comment:
아래의 코드로 개선해 보면 어떨까요?

```suggestion
        assertThat(LottoNumber.of(input)).isSameAs(LottoNumber.of(input));
```